### PR TITLE
Update ListCheckRunsOptions with new field AppID

### PR DESCRIPTION
**Pull request by rojanDinc**
#2231

### CODE REVIEW

File: github/checks.go
Comment (gmlewis) [inline github/checks.go:237]:
```suggestion
	AppID     *int64    `url:"app_id,omitempty"`     // Filters check runs by GitHub App ID.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @rojanDinc!
Just one change, please, then we should be ready for a second LGTM before merging.

Note that any other contributor to this repo is welcome to provide the second LGTM/approval after reviewing.

**Review by gmlewis — APPROVED** [review]
Thank you, @rojanDinc !
LGTM.
Merging after tests pass.

**Comment by gmlewis** [comment]
It looks like the tests failed.  Please make sure to run `go generate ./...` and push the results.
Also make sure your code is run through `go fmt` to make the linter happy.
See CONTRIBUTING.md for more info.

**Comment by rojanDinc** [comment]
Hi, I forgot to run tests locally once I changed `AppID`s type to `int64` but that problem should be resolved now.

File: github/checks.go
Comment (gmlewis) [inline github/checks.go:237]: Please run `gofmt` on this file and push the changes.

Comment (rojanDinc) [reply github/checks.go]: My bad. I've formatted the code hopefully everything is in place now.

**Review by gmlewis — APPROVED** [review]
Thank you, @rojanDinc !
LGTM.
Merging.

**Pull request merged**
